eng/pipelines/common/variables.yml

```diff
@@ -0,0 +1,3 @@
+variables:
```

Comment:
I might be missing something, but where is this imported?

Reply:
Well, it's imported in Libraries' `eng\pipelines\libraries\variables.yml`. But yeah, not used by Installer yet (or anywhere else) because we don't have a def set up to do official builds yet. Shuffling this out of the CoreFX-specific vars just lets me switch `${{ and(ne(variables['System.TeamProject'], 'public'), notIn(variables['Build.Reason'], 'PullRequest')) }}` to `isOfficialBuild` in this PR with clear intent where it *will* come from once I work on an official build.

Reply:
I see. I guess this file will be imported in runtime.yml for the official builds work under a global variable section just as `eng\pipelines\libraries\.azure-ci.yml` does for it's variables template.

We can leave that as-side for the official build effort.